deep-links.md

```diff
@@ -0,0 +1,54 @@
+# Deep links documentation
+
+Deep links allows to redirect a user from an email, sms or in-app notification directly to a specific page in the app.
+
+A deep link consists of the "app domain" which tells you what app to open and the actual deep link pointing to the page in the app you want to open.
+
+Combine the two into one link and you get a better user friendly experience.
```

Comment:
Might also be worth mentioning that some deep links has fallbacks (such as service reminders) to web, if the app is not installed.

Maybe a also add a "Fallback" column with "Yes" or "No" value in the deeplinks table further down?

Reply:
Yeah. I think booking is the only fallback we have at the moment